requirements_test_pre_commit.txt

```diff
@@ -1,6 +1,6 @@
 autoflake==1.4
```

Comment:
I'd really like to have the pre-commit configuration file as the only source of truth, every week for 4 weeks we had to accept 4/5 dependabot upgrades on pylint. We could have only one upgrade by the pre-commit bot. Maybe we could just add pylint/flake8 (because these tool do not have online autofix) and pre-commit in the test requirements ?

Reply:
I would be fine with removing `pyupgrade`, `autoflake`, and `black-disable-checker` here as they aren't integrated into IDEs and only called with `pre-commit`. Everything else has some use